packages/riipen-ui/src/components/Avatar.test.jsx

```diff
@@ -0,0 +1,167 @@
+    let error;
+    try {
+      mount(<Avatar />);
+    } catch (e) {
+      error = e;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const wrapper = mount(<Avatar />);
+
+      expect(wrapper.find("Avatar").props().size).toEqual("96px");
+      expect(wrapper.find("Avatar").props().variant).toEqual("circle");
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it("appends higher order values to default classes prop with withClass decorator", () => {
+      const wrapper = mount(<Avatar />);
+
+      expect(wrapper.find("Avatar").props().classes).toEqual([
+        "riipen",
+        "riipen-avatar"
+      ]);
+    });
+  });
+
+  describe("alt prop", () => {
+    it("sets inner class name given custom alt props", () => {
+      const altVariant = "Test String";
+
+      const wrapper = mount(<Avatar alt={altVariant} />);
+
+      expect(
+        wrapper
+          .find(".avatar")
+          .childAt(0)
+          .hasClass("inner")
+      ).toEqual(true);
+      expect(wrapper.find("Avatar").props().alt).toEqual(altVariant);
+    });
+  });
+
+  describe("children prop", () => {
+    it("displays given children", () => {
+      const initials = "ZZ";
+      const child = <div>{initials}</div>;
+
+      const wrapper = mount(<Avatar>{child}</Avatar>);
+
+      expect(
+        wrapper
+          .find(".inner")
+          .children()
+          .contains(child)
+      ).toEqual(true);
+      expect(
+        wrapper
+          .find(".inner")
+          .childAt(0)
+          .text()
+      ).toEqual(initials);
+    });
+  });
+
+  describe("classes prop", () => {
+    it("sets custom class", () => {
+      const classVariant = ["newClass"];
+
+      const wrapper = mount(<Avatar classes={classVariant} />);
+
+      expect(
+        wrapper
+          .find("Avatar")
+          .childAt(0)
+          .hasClass(classVariant[0])
+      ).toEqual(true);
+    });
+  });
+
+  describe("size prop", () => {
+    it("sets valid custom size", () => {
+      const sizeVariant = "50px";
+
+      const wrapper = mount(<Avatar size={sizeVariant} />);
+
+      expect(wrapper.find("Avatar").props().size).toEqual(sizeVariant);
+    });
+
+    it("sets valid border size above 70", () => {
+      const sizeVariant = "75px";
+
+      const wrapper = mount(<Avatar size={sizeVariant} />);
+
+      expect(
+        wrapper
+          .find("Avatar")
+          .instance()
+          .getBorderSize()
+      ).toEqual("4px");
+    });
+
+    it("sets valid border size below 70", () => {
+      const sizeVariant = "65px";
+
+      const wrapper = mount(<Avatar size={sizeVariant} />);
+
+      expect(
+        wrapper
+          .find("Avatar")
+          .instance()
+          .getBorderSize()
+      ).toEqual("2px");
+    });
+  });
+
+  describe("src prop", () => {
+    it("sets image class name given custom alt and src props", () => {
```

Comment:
Nice this is what I was trying to explain up above with the `alt` tests.

I think we don't need the `alt` here if we make the changes I proposed above. This tests should simply be "renders an img tag". See test below for how to update.